src/base-apis.js

```diff
@@ -891,18 +891,29 @@ MatrixBaseApis.prototype.sendStateEvent = function(roomId, eventType, content, s
 /**
  * @param {string} roomId
  * @param {string} eventId
+ * @param {string=} txnId     transaction id. One will be made up if not
```

Comment:
I'd mark this `@param {string} [txnId]` now

Reply:
oh is it since that is optional

Reply:
fixed :)